sdk/src/main/java/com/mapbox/maps/MapInitOptions.kt

```diff
@@ -36,39 +36,19 @@ data class MapInitOptions constructor(
    * Static methods
    */
   companion object {
-    /**
-     * The default cache size, which is 50MB
-     */
-    const val DEFAULT_CACHE_SIZE = 1024 * 1024 * 50L
-
     /**
      * Get a default [ResourceOptions] with token from default [CredentialsManager]
      * @property context the context of the application.
      */
-    fun getDefaultResourceOptions(context: Context): ResourceOptions = ResourceOptions.Builder()
-      .accessToken(CredentialsManager.default.getAccessToken(context))
-      .cachePath("${context.filesDir.absolutePath}/$DATABASE_NAME")
-      .cacheSize(DEFAULT_CACHE_SIZE) // 50 mb
-      .build()
+    fun getDefaultResourceOptions(context: Context): ResourceOptions =
+      ResourceOptionsManager.getDefault(context).resourceOptions
```

Comment:
Concerning me a bit that `ResourceOptions` here are built not alike default map options in `getDefaultMapOptions` below.

Reply:
this is to allow taking the shared default ResourceOptions from the ResourceOptionsManager, we don't have such setup for MapOptions.

Reply:
Also ResourceOptions is used for OfflineManager, Snapshotter etc, which has the requirement that the tile store must be consistent with the one used to initialise the MapView.